### PR TITLE
Xuchen's edits

### PR DESCRIPTION
**Pull request by xuchen**

### CODE REVIEW

File: config/_default/params.toml [hunk 1]
```diff
@@ -12,8 +12,8 @@ docsVersion = "0.1"
 ## Open Graph
 images = ["doks.png"]
 ogLocale = "en_US"
-domainTLD = "doks.netlify.app"
-titleHome = "Doks Theme"
+domainTLD = "wiki.seasalt.ai"
+titleHome = "Seasalt.ai"
```
Comment (xuchen) [inline config/_default/params.toml:16]: @amy17519 inside this config, could you please LMK:

1. what does L13 `images = ["doks.png"]` do? can you replace it with ours? I can see the example at https://getdoks.org/doks.png but don't understand what it's meant for.
2. I can't find the file on L31 `schemaLogo = "logo-seasalt.png"`

**Comment by xuchen** [comment]
@amy17519 could you pls help me fix the large blank in the footer? 
image

feel free to directly edit on the branch, thanks!

Since this is publishing from `gh_pages`, should I set the main branch from `main` to  `gh_pages`? So the publishing process changes from `my_branch --> main --> gh_pages` to `my_branch --> ph_pages`

File: config/_default/params.toml (hunk 1 above)
Comment (amy17519) [reply config/_default/params.toml:16]: I haven't explored all the features beyond the webpage itself. These options are for Open Graph and JSON-LD (https://getdoks.org/docs/reference-guides/seo/#open-graph--twitter-cards). Should be helpful for SEO?! I will update these fields to be related to Seasalt.

**Review by amy17519 — APPROVED** [review]

**Comment by amy17519** [comment]
@xuchen Please keep the `main` branch as the main branch. The publishing process should be `push to main` -> `github actions build the new webpages from changes in main branch` -> `github actions pushes the new version -- only the static files-- to gh-pages branch`

It's up to you whether you want to enforce PR review before merging to `main`. Right now I can push to `main` directly. However, we need both `main` and `gh-pages` + [actions-gh-pages](https://github.com/peaceiris/actions-gh-pages) for this to work.

**Comment by xuchen** [comment]
> @xuchen Please keep the `main` branch as the main branch. The publishing process should be `push to main` -> `github actions build the new webpages from changes in main branch` -> `github actions pushes the new version -- only the static files-- to gh-pages branch`
> 
> It's up to you whether you want to enforce PR review before merging to `main`. Right now I can push to `main` directly. However, we need both `main` and `gh-pages` + [actions-gh-pages](https://github.com/peaceiris/actions-gh-pages) for this to work.

I see! I forgot about github actions automation. thanks for reminding of that. Coiuld you pls add this to `README.md`  in a section called `How to Develop`? Thanks!

**Pull request merged**